example/client.rb

```diff
@@ -212,47 +213,63 @@ def create_and_fund_transaction_example
     end
   end
 
+  ResponseStruct = Struct.new(:headers, :body, :url)
+
   def webhook_parse_example
-    webhook_content = <<-JSON
-      {
-        "webhook": "85d11cf4-d4d6-46e4-ab7d-91355e80392a",
-        "event": "transaction.created",
-        "object": {
-          "input_amount": 21.59,
-          "input_currency": "GBP",
-          "id": "cd390e83-a9de-404d-a660-4da0f1342c04",
-          "metadata": {},
+    headers = <<-JSON
```

Comment:
no need to have this as a JSON string - we can expect that the user will get the headers in a nice format from the library used.

Reply:
Also we only need the nonce, key and secret here, not the rest